pilot/cmd/pilot-agent/options/options.go

```diff
@@ -134,4 +134,8 @@ var (
 	exitOnZeroActiveConnectionsEnv = env.RegisterBoolVar("EXIT_ON_ZERO_ACTIVE_CONNECTIONS",
 		false,
 		"When set to true, terminates proxy when number of active connections become zero during draining").Get()
+
+	DualStackEnv = env.RegisterBoolVar("ISTIO_AGENT_DUAL_STACK",
```

Comment:
what's this for

Reply:
need to enable dual stack feature for pilot-agent. (One is for pilot, the other is for pilot-agent)